src/components/Form.js

```diff
@@ -74,42 +105,46 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
+  onChange = (formData, options = {validate: false}) => {
```

Comment:
Note that you can totally land the patch using the regular Promise API and we shall migrate the code to async/await in a next PR if that worries you